source/install.txt

```diff
@@ -161,20 +161,28 @@ To download and install |compass|, select your operating system:
          .. tab:: Debian
             :tabid: debian
 
-            1. Download |compass|
+            1. Download |compass|.
 
                .. code-block:: shell
 
                   wget https://downloads.mongodb.com/compass/mongodb-compass_{+current-version+}_amd64.deb
                
 
-            #. Install |compass|
+            #. Install |compass|.
+
+               .. code-block:: shell
+
+                  sudo apt install mongodb-compass_{+current-version+}_amd64.deb
+
+               If your Linux distribution does not support ``apt``,
```

Comment:
All relevant Linux distros that support `dpkg` also support `apt`. This should probably say something along the lines of

```suggestion
               If your Linux distribution does not support installing local ``.deb`` files through ``apt``,
```

Reply:
Done!